packages/client/src/pages/common/profile/profile-dialog.tsx

```diff
@@ -77,6 +90,33 @@ const ProfileDialog = ({ open, onClose }: ProfileDialogProps) => {
 
   const fieldStyle = { margin: "0.5rem 0rem", color: "#595959" };
 
+  useEffect(() => {
+    const getUserProfile = async () => {
+      const userProfile = await userService.getUserProfile(userId);
```

Comment:
Wait why not just update the redux state while we're at it.

Reply:
will go with userId change

Reply:
Empty dependency array would only run on mount/unmount, need to have `userId` in dependency array if that's the case.